test_conformance/vulkan/procs.h

```diff
@@ -0,0 +1,23 @@
+/******************************************************************
+//
+//  OpenCL Conformance Tests
+// 
+//  Copyright:	(c) 2008-2011 by Apple Inc. All Rights Reserved.
+//
+******************************************************************/
+
+#include "../../test_common/harness/mt19937.h"
+
+#pragma mark -
+#pragma Misc tests
```

Comment:
Why the pragma?

Reply:
Was borrowed from some other tests. Removed.